src/form.js

```diff
@@ -959,6 +959,9 @@ define([
             var value = exporter.generateExportTSV(this);
             this.vellum.afterSerialize();
             return value;
+        },
+        isCaseReference: function (path) {
+            return /instance\('casedb'\)/.test(path) ? true : false;
```

Comment:
nitpicks: not sure if it matters in practice, but would be more robust to match ` /instance\(['"]casedb['"]\)/` in case double quotes were ever used. also ` ? true : false` is superfluous.